win32/Makefile.a64

```diff
@@ -23,7 +23,13 @@ AR = lib
 RC = rc
 CP = copy /y
 CFLAGS  = -nologo -MD -W3 -O2 -Oy- -Zi -Fd"zlib" $(LOC)
-WFLAGS  = -D_CRT_SECURE_NO_DEPRECATE -D_CRT_NONSTDC_NO_DEPRECATE -DUNALIGNED_OK -DUNALIGNED64_OK -D_ARM64_WINAPI_PARTITION_DESKTOP_SDK_AVAILABLE=1
+WFLAGS  = \
+	-D_CRT_SECURE_NO_DEPRECATE \
+	-D_CRT_NONSTDC_NO_DEPRECATE \
+	-DUNALIGNED_OK \
+	-DUNALIGNED64_OK \
+	-D_ARM64_WINAPI_PARTITION_DESKTOP_SDK_AVAILABLE=1 \
```

Comment:
Are trailing '\\' okay in Makefiles?

Reply:
I suppose if it CI succeeds we will know

Reply:
The idiom is to add a # on the following line.  Otherwise somebody might put something there and have the line continued by accident.

Reply:
Good idea

Reply:
The other nice thing is it's easier to keep them sorted alphabetically this way.

mmjjjjj!'msort   :-)

Reply:
Fixed